content/rs/administering/security/ldap-integration.md

```diff
@@ -62,7 +59,7 @@ ldap_search_base: ou=coolUsers,dc=company,dc=com
 
 # Specify a search filter. The value for the configuration option (%u)
 # should correspond to parameters (uid) specific for your installation.
```

Comment:
should the "uid" in the brackets not be sAMAccountName as well?

Reply:
@danni-m Do you agree with Roey?

Reply:
Makes sense `uid` in this context refers to the ldap_filter parameter.

Reply:
@danni-m Please approve the commit: https://github.com/RedisLabs/redislabs-docs/pull/181/commits/b0e558c638ccff13f58b4bc328cc44c29508f5e2